data/helpers.d/psql

```diff
@@ -128,7 +126,7 @@ ynh_psql_dump_db() {
 ynh_psql_create_user() {
 	local user=$1
 	local psql=$2
-	ynh_psql_execute_as_root --sql="CREATE USER $user WITH PASSWORD '$pwd'"
+	ynh_psql_execute_as_root --sql="CREATE USER $user WITH PASSWORD $pwd"
```

Comment:
Uh what's this `psql` variable ? Shouldn't it be `pwd` ?

Reply:
I'll change that! Thx